src/main/scala/org/apache/spark/sql/catalyst/optimizer/RangerSparkAuthorizerExtension.scala

```diff
@@ -75,6 +80,48 @@ case class RangerSparkAuthorizerExtension(spark: SparkSession) extends Rule[Logi
     }
   }
 
+  private def authorizeSetCommand(s: SetCommand) = {
+    s.kv match {
+      case Some(kv) => {
+        /**
+         * This is needed since Spark 2.4, which included a new configuration that allows to exclude some
+         * rules on the Optimizer, which would deactivate all Ranger's authorization.
+         *
+         * In the future, if Spark allows to configure the rules that can not be excluded, this check will not
+         * be necessary. At the moment, the list of non excludable rules is hardcoded on Spark's optimizer.
+         */
+        if (kv._1.equalsIgnoreCase("spark.sql.optimizer.excludedRules") && isRangerExtension(kv._2)) {
```

Comment:
Can you check case-sensitive mode?  I thinks spark configs are case sensitive for both k and v.

Reply:
I've changed this, but I'm not 100% sure if we should do that. I think that it's safer to compare case insensitive, just in case that there's any change of behavior on a following Spark version.

Reply:
I understand what you concern, but AFAIK, spark configs are still case sensitive in the coming spark 3.x. And case-insensitive here is a bit overkilling to me as it may affect variable substitution in spark.

Reply:
I'm ok if you think it's safe enough 👍